ui/src/app/views/cohort-builder-placeholder/component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {TestBed, async} from '@angular/core/testing';
```

Comment:
Incidentally, I ran across an article which says:

> Note that we're utilizing the async test method here. This allows us to create our components for testing, which is an asynchronous operation, in a way that executes our tests only after the asynchronous operations have completed.

https://semaphoreci.com/community/tutorials/testing-routes-in-angular-2

So I think `async` is not doing anything parallel, it's just helping us manage async setup. (I wonder if wrapping `async` around other stuff where we don't care about specific timing intervals would simplify anything.)